### PR TITLE
ID22817: added Agrieos description

### PR DESCRIPTION
**Pull request by sviridovartem**

### CODE REVIEW

File: webviewer/public/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="description" content="Geoap" />
+    <meta name="description" id="description" content="SoilMate" />
```
Comment (adolgarev) [inline webviewer/public/index.html:6]: Why Soilmate here? We have a different branch for soilmate.

File: webviewer/public/index.html [hunk 2]
```diff
@@ -64,14 +64,16 @@
         var manifest = document.getElementById('manifest');
         var favicon32 = document.getElementById('favicon32');
         var favicon16 = document.getElementById('favicon16');
+        var description = document.getElementById('description');
 
         var domainMappings = {
-          'agrieos.in': {
+          'localhost': {
```
Comment (adolgarev) [inline webviewer/public/index.html:70]: localhost?

**Review by adolgarev — COMMENTED** [review]
Actually this is a good opportunity to get rid of Soilmate branch. We can have now geoap branding by default, soilmate branding for portal.soilmate.ai, and agrieos for agrieos.in

File: webviewer/src/components/WelcomeWindow/WelcomeWindow.js [hunk 3]
```diff
@@ -16,9 +16,12 @@ export const WelcomeWindow = ({ onClose }) => {
   const handleClick = e => setIsChecked(e.target.checked);
   const handleClose = () => onClose(isChecked);
 
+  const domain = window.location.hostname;
+  const title = domain === 'localhost' ? 'Agrieos' : 'SoilMate';
```
Comment (adolgarev) [inline webviewer/src/components/WelcomeWindow/WelcomeWindow.js:20]: Please collect all such things which decide upon domain name what to show in one place. Thus we won't need to look for 'Agrieos' and 'agrieos.in' in many files

Comment (sviridovartem) [reply webviewer/src/components/WelcomeWindow/WelcomeWindow.js]: fixed

**Review by adolgarev — APPROVED** [review]

**Pull request merged**